tmt/steps/prepare/install.py

```diff
@@ -0,0 +1,68 @@
+import fmf
+import tmt
+import click
+
+class PrepareInstall(tmt.steps.prepare.PreparePlugin):
+    """
+    Install packages on the guest
+
+    Example config:
+
+    prepare:
+        how: install
+        copr: psss/tmt
+        package: tmt-all
+
+    Use 'order' attribute to select in which order preparation should
+    happen if there are multiple configs. Default order is '50'.
+    Default order of required packages installation is '70'.
+    """
+
+    # Supported methods
+    _methods = [tmt.steps.Method(name='install', doc=__doc__, order=50)]
+
+    @classmethod
+    def options(cls, how=None):
+        """ Prepare command line options """
+        return [
+            click.option(
+                '-p', '--package', metavar='NAME', multiple=True,
```

Comment:
this is not sufficient, we need to be able to install all packages from copr repo also ... 

See how cruncher does this ...

Maybe it can be a separate issue address outside of this PR

Reply:
Yes, let's make it a new issue. I've added just a very simple `install` plugin implementation just to get started.